src/main/target/REVO/target.h

```diff
@@ -308,11 +311,16 @@
 #endif
 
 #define USE_ADC
-#if !defined(PODIUMF4)
+#if defined(PODIUMF4)
```

Comment:
This seems to be getting needlessly complicated. This will achieve the same result:

```
	#if !defined(PODIUMF4)
￼￼	#define CURRENT_METER_ADC_PIN   PC1
￼	#define VBAT_ADC_PIN            PC2
￼￼	#else
￼	#define VBAT_ADC_PIN            PC3
￼￼	#endif

￼	#if defined(ELINF405)
	#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
￼	#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC 
	#endif 
```

Reply:
I agree. It was fixed.